### PR TITLE
Fix #574 Removed storybooks from expandable-search-bar

### LINKED ISSUE #574: Remove storybook from expandable-search-bar package
**Task Details**

> We don't use storybook for demoing `expandable-search-bar` so remove it as a dependency

**Verification**

> Storybook, its scripts, and dependencies are removed from package.json and instructions removed from the readme

**Stakeholders**

> @jbuckner

Replaces https://github.com/internetarchive/iaux/issues/534

### PR DESCRIPTION
**Pull request by soham4abc**
**Description**

> Removes `storybook` from `expandable-search-bar` 
fix #574 
`Closes #574`.

**Technical**

> Storybooks scripts removed from `package.json` and `README`.

### CODE REVIEW

File: packages/expandable-search-bar/package.json [hunk 1]
```diff
@@ -32,25 +32,21 @@
...
-    "storybook:build": "build-storybook -o _site -s storybook-static"
+    "test:bs": "karma start karma.bs.config.js --compatibility all --coverage"
+    
   },
   "dependencies": {
     "lit-element": "^2.2.1",
     "lit-html": "^1.0.0"
   },
   "devDependencies": {
-    "@open-wc/demoing-storybook": "^0.3.0",
```
Comment (jbuckner) [inline packages/expandable-search-bar/package.json:45]: There should also be an update to the `yarn.lock` file when removing these. Did you run `yarn remove` or just delete these lines?

Comment (soham4abc) [reply packages/expandable-search-bar/package.json:45]: yarn.lock updated

File: packages/expandable-search-bar/package.json [hunk 2]
```diff
@@ -32,25 +32,21 @@
     "test:prune-snapshots": "karma start --prune-snapshots",
     "test:compatibility": "karma start --compatibility all --coverage",
     "test:compatibility:watch": "karma start --compatibility all --auto-watch=true --single-run=false",
-    "test:bs": "karma start karma.bs.config.js --compatibility all --coverage",
-    "site:build": "npm run storybook:build",
-    "storybook": "start-storybook -p 9001",
-    "storybook:build": "build-storybook -o _site -s storybook-static"
+    "test:bs": "karma start karma.bs.config.js --compatibility all --coverage"
+
```
Comment (jbuckner) [inline packages/expandable-search-bar/package.json:36]: Could you remove the extra blank lines here and in `devDependencies`?

Comment (soham4abc) [reply packages/expandable-search-bar/package.json]: It's done as per request. Any more changes?

**Review by jbuckner — APPROVED** [review]
Looks good! Thanks for the contribution!

**Pull request merged**